go/oasis-test-runner/oasis/fixture.go

```diff
@@ -118,6 +119,8 @@ type ValidatorFixture struct {
 
 	Entity int `json:"entity"`
 
+	LogWatcherHandlerFactories []log.WatcherHandlerFactory `json:"log_fac"`
```

Comment:
Fixtures are supposed to be serializable but factories currently are not. Either set `json:"-"` or make the factories serializable (probably something like having a registry of factory names and using the name on serialization and recreating from the registry on deserialization).

Also in other fixture structs.

Reply:
I've set `json:"-"`.